server/methods/getUsersOfRoom.js

```diff
@@ -23,7 +23,10 @@ Meteor.methods({
 		};
 
 		const map = (record) => {
-			return record._user.username;
+			return {
```

Comment:
This is an API change, can you describe on HISTORY.md that will change from an array of strings to an array of objects?